src/utils/admin-utils.ts

```diff
@@ -67,10 +60,10 @@ export const getChartAdminBoundaryParams = (
   );
 
   const adminBoundaryParams: AdminBoundaryParams = {
-    title,
     boundaryProps,
-    serverParams: { layerName: serverLayerName, url },
-    chartType,
+    serverLayerName,
+    url,
+    id: levels[levels.length - 1].id,
```

Comment:
Let's create a util function for this (`levels[levels.length - 1].id`) since we are using it throughout the app and it could be error-prone if we were to change the order somewhere